s3store/s3store.go

```diff
@@ -517,6 +501,33 @@ func (store S3Store) ConcatUploads(dest string, partialUploads []string) error {
 	return store.FinishUpload(dest)
 }
 
+func (store S3Store)ListAllParts(id string) (parts []*s3.Part, err error) {
```

Comment:
No, we would need to test it through GetInfo and FinishUpload since the test cases are in a separate package.